src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.cs

```diff
@@ -0,0 +1,359 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using BizHawk.Common;
+using BizHawk.Emulation.Common;
+using BizHawk.Emulation.Cores.Properties;
+using BizHawk.Emulation.Cores.Waterbox;
+
+namespace BizHawk.Emulation.Cores.Computers.DOS
+{
+	[PortedCore(
+		name: CoreNames.DOSBox,
+		author: "Jonathan Campbell et al.",
+		portedVersion: "2025.02.01 (324193b)",
+		portedUrl: "https://github.com/joncampbell123/dosbox-x",
+		isReleased: false)]
+	public partial class DOSBox : WaterboxCore
+	{
+		private static readonly Configuration DefaultConfig = new Configuration
+		{
+			SystemId = VSystemID.Raw.DOS,
+			MaxSamples = 8 * 1024,
+			DefaultWidth = LibDOSBox.VGA_MAX_WIDTH,
+			DefaultHeight = LibDOSBox.VGA_MAX_HEIGHT,
+			MaxWidth = LibDOSBox.SVGA_MAX_WIDTH,
+			MaxHeight = LibDOSBox.SVGA_MAX_HEIGHT,
+			DefaultFpsNumerator = LibDOSBox.VIDEO_NUMERATOR_DOS,
+			DefaultFpsDenominator = LibDOSBox.VIDEO_DENOMINATOR_DOS
+		};
+
+		private readonly List<IRomAsset> _roms;
+		private const int _messageDuration = 4;
+
+		// Drive management variables
+		private bool _nextFloppyDiskPressed = false;
+		private bool _nextCDROMPressed = false;
+		private List<IRomAsset> _floppyDiskImageFiles = new List<IRomAsset>();
+		private List<IRomAsset> _CDROMDiskImageFiles = new List<IRomAsset>();
+		private int _floppyDiskCount = 0;
+		private int _CDROMCount = 0;
+		private int _currentFloppyDisk = 0;
+		private int _currentCDROM = 0;
+		private string GetFullName(IRomAsset rom) => rom.Game.Name + rom.Extension;
+
+		public override int VirtualWidth => LibDOSBox.SVGA_MAX_WIDTH;

```

Comment:
`VirtualWidth` represents how the frame buffer should be stretched by forntend to look authentic to how the user saw the final thing back in the day. For IBM PC that means it's just a 4:3 monitor. Now since height can't be stretched (because it represents scanlines), we should just tell frontend to resize width to be  4/3 of height.

Reply:
Thanks, I made the corresponding change and it looks correct now